mlos_bench/mlos_bench/config/__init__.py

```diff
@@ -197,9 +197,40 @@
      "current_dir": "$PWD",
      "some_expanded_var": "$some_var: $experiment_id",
      "location": "eastus",
+
+     // This can be specified in the CLI config or the globals config
+     "tunable_params_map": {
+        // a map of tunable_params variables to their covariant group names
+        "environment1_tunables": [
+            "covariant_group_name",
+            "another_covariant_group_name"
+        ],
+        "environment2_tunables": [
+            // empty list means no tunables
+            // are enabled for this environment
+            // during this experiment
+            // (e.g., only use defaults for this environment)
+        ],
    }
 
-There are additional details about variable propagation in the
+Users can have multiple global config files, each specified with a ``--globals``
+CLI arg or ``"globals"`` CLI config property.
+
+At runtime, parameters from these files will be combined into a single
+dictionary, in the order they appear, and pushed to the root
+:py:class:`Environment <mlos_bench.environments>`.
+
+Any global or :py:class:`~.Environment` parameter can also be overridden from
+the command line, by simply specifying ``--PARAMETER_NAME PARAMETER_VALUE``.
+
+Another common use of global config files is to store sensitive data (e.g.,
+passwords, tokens, etc.) that should not be version-controlled.
+
+This way, users can keep their experiment-specific parameters separately from
+the Environment configs making them more reusable.
```

Comment:
Moved the explanation of globals from mlos_bench.environments to mlos_bench.config